### PR TITLE
cSpell fixes for instrumentation pages: java

### PR DESCRIPTION
**Pull request by svrnm**
cSpell fixes & ignore list updates

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]

**Review by chalin — APPROVED** [review]

File: content/en/docs/instrumentation/java/automatic/agent-config.md [hunk 1]
```diff
@@ -4,7 +4,12 @@ linkTitle: Configuration
 weight: 10
 spelling:
   cSpell:ignore akka autoconfiguration Autoconfiguration Dotel HSET javaagent
-  cSpell:ignore serverlessapis Servlet servlet
+  cSpell:ignore serverlessapis Servlet servlet Customizer classloaders logback
```
Comment (chalin) [inline content/en/docs/instrumentation/java/automatic/agent-config.md:7]: Do we actually want both Servlet and servlet in this list? Only "servlet" should be enough, right?

Comment (svrnm) [reply content/en/docs/instrumentation/java/automatic/agent-config.md:7]:
```suggestion
  cSpell:ignore serverlessapis servlet Customizer classloaders logback
```

File: content/en/docs/instrumentation/java/automatic/agent-config.md [hunk 2]
```diff
@@ -4,7 +4,12 @@ linkTitle: Configuration
 weight: 10
 spelling:
   cSpell:ignore akka autoconfiguration Autoconfiguration Dotel HSET javaagent
-  cSpell:ignore serverlessapis Servlet servlet
+  cSpell:ignore serverlessapis servlet Customizer classloaders logback
```
Comment (breedx-splk) [inline content/en/docs/instrumentation/java/automatic/agent-config.md:7]: I notice start-cap on `Customizer` but I suppose case is probably not important.

Comment (svrnm) [reply content/en/docs/instrumentation/java/automatic/agent-config.md:7]: it's like "if upper case, the case counts, if lowercase it's both upper & lower case", so yes we are good here :)

**Pull request merged**